core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -355,6 +356,8 @@ object TopicCommand extends Logging {
 
     val forceOpt = parser.accepts("force", "Suppress console prompts")
 
+    val excludeInternalTopic = parser.accepts("exclude-internal", "exclude the intenral topics when lising the topic name. the internal topic will be listed by default")
```

Comment:
- Does this also work for --describe? If so, it should be stated in here.
- There is a typo in `internal`
- There are excess `the`s. 
- Can you please capitalize the beginning of the second sentence?

Reply:
Thanks for the reviews! I will address your comment after completing the KIP flow.